### ISSUE TITLE
No support to link user claim script to Identity Provider

### ISSUE THREAD

**Issue by ashwinnirantar** [appgate/terraform-provider-appgatesdp#246]
Thank you for the recent version upgrade where support for `user_claim_scripts` was added. 

When I tried to create the user claim script and link it to our Identity Provider, I realized that `user_scripts` in https://registry.terraform.io/providers/appgate/appgatesdp/latest/docs/resources/saml_identity_provider is not supported right now. Terraform complained about it so I checked the source code and was not able to find it. 

Let me know if I am missing something. 

Thank you.

**Comment by danijeel** [comment]
Thanks for reporing this issue, Looks like we missed to add that attribute. Will fix it in the upcoming release.

**Comment by ashwinnirantar** [comment]
@danijeel Thank you for the quick response as always, really appreciate your support

**Comment by danijeel** [comment]
Its now available in [v0.8.13](https://registry.terraform.io/providers/appgate/appgatesdp/0.8.13)
example usage:

```hcl

resource "appgatesdp_user_claim_script" "custom_script" {
  name       = "user claim script"
  notes      = "This object has been created for test purposes."
  expression = "return {'posture': 25};"
  tags = [
    "developer",
    "api-created"
  ]
}

resource "appgatesdp_saml_identity_provider" "saml_test_resource" {
  depends_on = [
    appgatesdp_user_claim_script.custom_script
  ]
  name           = "test saml provider"
  admin_provider = true
  ip_pool_v4     = data.appgatesdp_ip_pool.ip_v4_pool.id
  ip_pool_v6     = data.appgatesdp_ip_pool.ip_v6_pool.id
  dns_servers = [
    "8.8.8.8",
  ]
  dns_search_domains = [
    "internal.company.com"
  ]
  redirect_url = "https://saml.company.com"
  issuer       = "http://adfs-test.company.com/adfs/services/trust"
  audience     = "Company Appgate SDP"

  user_scripts = [
    appgatesdp_user_claim_script.custom_script.id
  ]

  // ..
}

```
